Sources/Tabman/TabmanBar/Styles/Abstract/TabmanButtonBar.swift

```diff
@@ -45,22 +45,15 @@ internal class TabmanButtonBar: TabmanBar {
             
             focussedButton?.setTitleColor(self.selectedColor, for: .normal)
             focussedButton?.tintColor = self.selectedColor
+            focussedButton?.titleLabel?.font = self.selectedTextFont
             oldValue?.setTitleColor(self.color, for: .normal)
             oldValue?.tintColor = self.color
+            oldValue?.titleLabel?.font = self.textFont
         }
     }
     
-    public var textFont: UIFont = Appearance.defaultAppearance.text.font! {
-        didSet {
-            guard textFont != oldValue else {
-                return
-            }
-            
-            self.updateButtons(update: { (button) in
-                button.titleLabel?.font = textFont
-            })
-        }
-    }
```

Comment:
@moaible - Looks like this might be necessary so that the custom text is set when the view loads (before the tab is changed).